src/Authentication/AuthenticationManager.php

```diff
@@ -103,7 +104,7 @@ public function getAccount(RequestInterface $request, $cache = TRUE) {
 
     if (!$account) {
 
-      if ($this->plugins->count() && !$this->getIsOptional()) {
+      if (RestfulManager::isRestfulPath($request) && $this->plugins->count() && !$this->getIsOptional()) {
```

Comment:
``RestfulManager::isRestfulPath($request)`` -- how come this was invoked if this wasn't a restful path?

do we care if it's not a restful path? I for example already had a cases where I've manually invoked the authentication manager for non restful path.